docs/manual/en/introduction/Installation.html

```diff
@@ -10,149 +10,245 @@
 		</p>
 
 		<code>
-		npm install three
+			npm install --save three
 		</code>
 
+		<h2>Project structure</h2>
+
 		<p>
-			The package will be downloaded and installed. Then you're ready to import it in your code:
+			Every three.js project needs at least one HTML file to define the webpage, and a JavaScript file to run your three.js code. The structure and naming choices below aren't required, but will be used throughout this guide for consistency.
 		</p>
 
-		<code>
-		// Option 1: Import the entire three.js core library.
-		import * as THREE from 'three';
+		<ul>
+			<li>
+				<i>index.html</i>
+				<code>
+		&lt;!DOCTYPE html&gt;
+		&lt;html lang="en"&gt;
+			&lt;head&gt;
+				&lt;meta charset="utf-8"&gt;
+				&lt;title&gt;My first three.js app&lt;/title&gt;
+				&lt;style&gt;
+					body { margin: 0; }
+				&lt;/style&gt;
+			&lt;/head&gt;
+			&lt;body&gt;
+				&lt;script type="module" src="/main.js"&gt;&lt;/script&gt;
+			&lt;/body&gt;
+		&lt;/html&gt;
+				</code>
+			</li>
+			<li>
+				<i>main.js</i>
+				<code>
+import * as THREE from 'three';
 
-		const scene = new THREE.Scene();
+...
+				</code>
+			</li>
+			<li>
+				<i>public/</i>
+				<ul>
+					<li>
+						The <i>public/</i> folder is sometimes also called a "static" folder, because the files it contains are pushed to the website unchanged. Usually textures, audio, and 3D models will go here.
+					</li>
+				</ul>
+			</li>
+		</ul>
+
+		<p>
+			Now that we've set up the basic project structure, we need a way to run the project locally and access it through a web browser.
+		</p>
+
+		<h2>Option 1: Installing with a build tool</h2>
+
+		<h3>Development</h3>
+
+		<p>
+			Installing from npm and using a build tool is the recommended approach for most users — the more dependencies your project needs, the more likely you are to run into problems that the static hosting cannot easily resolve. With a build tool, importing local JavaScript files and npm packages should work out of the box, without import maps.
+		</p>
+
+
+		<ol>
+			<li>
+				Install [link:https://nodejs.org/ Node.js]. We'll need it to load manage dependencies and to run our build tool.
+			</li>
+			<li>
+				Install three.js and a build tool, [link:https://vitejs.dev/ Vite], using a [link:https://www.joshwcomeau.com/javascript/terminal-for-js-devs/ terminal] in your project folder. Vite will be used during development, but it isn't part of the final webpage. If you prefer to use another build tool, that's fine — we support modern build tools that can import [link:https://eloquentjavascript.net/10_modules.html#h_zWTXAU93DC ES Modules].
+				<code>
+# three.js
+npm install --save three
 
-
-		// Option 2: Import just the parts you need.
-		import { Scene } from 'three';
-
-		const scene = new Scene();
-		</code>
+# vite
+npm install --save-dev vite
```

Comment:
hmm why are you telling the user to install vite if you're running it with `npx` in the next command?

If we want to tell the user to install it globally maybe the steps can be like this:
2. Install three.js as a dependency
```
npm install --save three
```
3. Use a build tool...
```
npx vite
```

----------------

Otherwise, if we want to tell the user to install vite locally, we can tell him to first run:

```
npm create vite@latest
```
as explained in [the vite guide](https://vitejs.dev/guide/)

Reply:
My thoughts here were:

1. Vite should be installed as a project dependency
2. Didn't want to complicate this with explanations of `package.json` and `scripts` macros

I'm not a huge fan of `npm create vite`, because it generates more stuff that's outside the scope of the guide, might change outside our control, and doesn't work for users who are following along but are actually adding three.js to an existing project. Switching from CDN to Vite, for example.

I use `npx <foo>` pretty regularly to run a project dependency, it [uses local dependencies fine](https://www.npmjs.com/package/npx#description). I think that's OK to do here? Otherwise we have to guide them through modifying `package.json` and creating these macros, just seemed more confusing.

Reply:
Fair enough, didn't know npx would run local deps first.

Reply:
Added some asides to explain package.json, node_modules/, and npx — thanks!